src/category_theory/punit.lean

```diff
@@ -27,6 +27,12 @@ namespace of
 @[simp] lemma map_app {X Y : C} (f : X ⟶ Y) : (of.map f).app = λ _, f := rfl
 end of
 
+def star : C ⥤ punit.{w+1} := (const C).obj punit.star
+namespace star
+@[simp] lemma obj (X : C) : star.obj X = punit.star := rfl
```

Comment:
I think I would rather call these `star_obj` and `star_map` *outside* of the `star` namespace. In that way `star.obj` and `star.map` unambigously refer to the components of the functor `star`.

Reply:
Oops, yes.